support/cas-server-support-pac4j/src/main/java/org/apereo/cas/support/pac4j/config/Pac4jConfiguration.java

```diff
@@ -170,7 +170,8 @@ private void configureSamlClient(final Map<String, String> properties) {
         properties.put(PropertiesConfigFactory.SAML_SERVICE_PROVIDER_ENTITY_ID,
                 casProperties.getAuthn().getPac4j().getSaml().getServiceProviderEntityId());
         properties.put(PropertiesConfigFactory.SAML_SERVICE_PROVIDER_METADATA_PATH,
-                casProperties.getAuthn().getPac4j().getSaml().getServiceProviderEntityId());
+                casProperties.getAuthn().getPac4j().getSaml().getServiceProviderMetadataPath());
+        properties.put(PropertiesConfigFactory.SAML_DESTINATION_BINDING_TYPE, "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect");
```

Comment:
And it does not compile. I will move the `libraries.opensaml` from `runtime` to `compile` scope if you don't have any concern with that.